### ISSUE TITLE
[Bug] Accesible engine selected have less accessibility

### ISSUE THREAD

**Issue by Biki-das** [creativecommons/search#117]
## Description
The engines are currently focusable  via keyboard as done by pr #52 however the dashed outline does not seem to be a viable one for the focus , making it solid seems a more viable option

current status
![CC Search Portal — Mozilla Firefox 10_20_2022 12_36_18 PM](https://user-images.githubusercontent.com/72331432/196879666-dc8b9662-980b-49d6-bd5e-e1861d89c830.png)

Changes made after PR
![CC Search Portal — Mozilla Firefox 10_20_2022 12_37_17 PM](https://user-images.githubusercontent.com/72331432/196880348-c5bc2371-fa38-4f4d-bc28-97f2f6255e91.png)

**Comment by ImaJin14** [comment]
@Biki-das this issue has not been triage so submitting a PR is against contribution guidelines....

**Comment by Biki-das** [comment]
@possumbilities could you give some feedback on this issue?

**Comment by Biki-das** [comment]
@possumbilities i have closed the pr sorry for that , but could you give some feedback is it good to work on this one?

**Comment by ImaJin14** [comment]
@Biki-das as I earlier mentioned, PR #108 aims on making openverse the default search engine. Meaning it'll be made unique from the rest and even if another engine was to be selected it still has to stand out... Somehow this is a duplicate to #108

**Comment by possumbilities** [comment]
I think if anything the focus indicators should be set to the browser default wherever possible, and not explicitly set to a style. 

Meaning instead of setting the outline to something, it should likely be removed entirely and let the browser style the focus indicator outline with its own inherent defaults.

**Comment by ImaJin14** [comment]
@Biki-das are you still working on this....

If not I'll love to resolve it @possumbilities

**Comment by Biki-das** [comment]
> @Biki-das are you still working on this....
> 
> 
> 
> If not I'll love to resolve it @possumbilities 

Yup working on it

**Comment by possumbilities** [comment]
Resolved in new site merge: https://github.com/creativecommons/search/pull/206
